lib/modules/keyboardNav.js

```diff
@@ -1078,19 +1074,36 @@ module.toggleChildren = function() {
 	click(thisToggle);
 };
 
-module.toggleExpando = function() {
-	const selected = SelectedEntry.selectedThing();
-	if (!selected) return;
+function toggleExpandos(scrollOnExpanding) {
```

Comment:
This function might make more sense in showImages.js, perhaps with the signature `toggleExpandos(thing, scrollOnExpanding)` or `toggleExpandos(expandos, scrollOnExpanding)`. But I don't feel strongly about this.